app/controllers/notification/wastes_controller.rb

```diff
@@ -67,6 +67,43 @@ def destroy
     end
   end
 
+  def ical_export
+    address = Address.joins(:waste_location_types).where(ical_export_params.slice(:street, :city, :zip)).first
+    full_address = "#{address.street}, #{address.zip} #{address.city}"
+    raise "Address not found" if address.blank?
+
+    waste_types = StaticContent.find_by(name: "wasteTypes").try(:content)
+    waste_types = JSON.parse(waste_types) if waste_types.present?
+    return if waste_types.blank?
+
+    @cal = Icalendar::Calendar.new
+    @cal.append_custom_property("NAME", "Abfallkalender")
+    @cal.append_custom_property("X-WR-CALNAME", "Abfallkalender")
+    @cal.append_custom_property("DESCRIPTION", "Alle Abholtermine der Müllabfuhr für #{full_address}")
+    @cal.append_custom_property("X-WR-CALDESC", "Alle Abholtermine der Müllabfuhr für #{full_address}")
```

Comment:
resultiert das in eine bestimmte version von ical? müssen wir das festhalten? in der beschreibung des PRs steht was von `VERSION: 2.0`. gibt es hier dinge zu beachten, falls wir anpassungen machen in zukunft?

Reply:
ich denke nicht das es unterschiedliche Versionen von iCal gibt die gleichzeitig im Umlauf sind. 
Dieses Syntax ist nur der Workaround um über das Gem "Icalendar::Calendar" die entsprechenden Attribute in dem generierten iCal zu erzeugen. 

Die Felder selbst gibt es in der ICal Spezifikation unabhängig davon.